### DISCUSSION TITLE
API's for Chargebacks

### DISCUSSION THREAD

**Discussion by Ja50nRu550** [amzn/selling-partner-api-models#623] [Ideas] [answered]
Companies both big and small need to centralize the chargeback data and automate the data aquisition for use in internal prescriptive and predictive analytics. For those of us with multiple subsidiaries, this would eliminate the need to manually login to each vendor central account, download the data, and upload it to our internal data warehouse. Also, for the times where there are limitations based on the number of records and having to split and then re-join data sets is error prone.

**Comment by krithic15** [comment, marked as answer]
Hello @Ja50nRu550,
Thank you for your suggestion. We have submitted your feature request to the appropriate team.
Thanks,
Krithi, Solutions Architect
Selling Partner Developer Services
